lib/omnibus/fetchers/net_fetcher.rb

```diff
@@ -125,6 +125,11 @@ def checksum
       source[digest_type]
     end
 
+    # Set to false to extract archives into the root source_dir (default).
+    # Set to true to always extract underneath project_dir
+    #
+    attr_accessor :isolate_extraction
```

Comment:
Why do we need this extra bit of configuration? Shouldn't we always isolate extraction? It seems that behavior would be fully backward compatible per any public APIs/DSL we current expose.